<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginManageServersWindow</class>
 <widget class="QWidget" name="LoginManageServersWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushAddServer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Neu</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listConnections">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>101</width>
     <height>221</height>
    </rect>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushDeleteConnection">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>LÃ¶schen</string>
   </property>
  </widget>
  <widget class="QWidget" name="widgetEditConnection" native="true">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>311</width>
     <height>291</height>
    </rect>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="labelName">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="inputName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelHostname">
       <property name="text">
        <string>Hostname:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="inputHostname"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelStandard">
       <property name="text">
        <string>Standard:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="checkboxStandard"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="portLabel">
       <property name="text">
        <string>Port:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="inputPort"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushSave">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Speichern</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushCancel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Abbrechen</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
